src/ResourceManagement/Batch/Domain/BatchAccount/Definition/IDefinition.cs

```diff
@@ -2,19 +2,13 @@
 // Licensed under the MIT License. See License.txt in the project root for license information.
 namespace Microsoft.Azure.Management.Batch.Fluent.BatchAccount.Definition
 {
-    using Microsoft.Azure.Management.ResourceManager.Fluent.Core.ResourceActions;
-    using Microsoft.Azure.Management.Storage.Fluent;
-    using Microsoft.Azure.Management.Batch.Fluent;
-    using Microsoft.Azure.Management.ResourceManager.Fluent.Core.Resource.Definition;
-    using Microsoft.Azure.Management.Batch.Fluent.Application.Definition;
-    using Microsoft.Azure.Management.ResourceManager.Fluent.Core.GroupableResource.Definition;
-
     /// <summary>
     /// The stage of a Batch account definition allowing the adding of a Batch application or creating the Batch account.
     /// </summary>
     public interface IWithCreateAndApplication :
-        Microsoft.Azure.Management.Batch.Fluent.BatchAccount.Definition.IWithCreate,
```

Comment:
What is the reason to remove the full qualified name?

Reply:
It's unnecessary to keep full qualified name here. This namespace is from IDefinition.cs itself.